olive/passes/onnx/transformer_optimization.py

```diff
@@ -76,6 +76,9 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
             "force_fp32_ops": PassConfigParam(
```

Comment:
could we use op_block_list/node_block_list as the parameters so that the transformer optimization could be more generic?

Reply:
This will break existing configs that use `force_fp32_ops`

Reply:
we could add a TODO and change force_fp32_nodes to node_block_list

Reply:
@trajepl / @jambayk , it seems force_fp32_nodes is used as node block list. I prefer the name to be consistent with that of transformer optimization. what's you thought?

Reply:
I think in the context of the transformer optimization pass, `force_fp32_ops/nodes` make more sense. Otherwise, the names `op/node_block_list` don't give any information about their uses. 
This conflict is present because we are using this pass to do the actual "transformer optimization" and "float_to_float16" steps. `op/node_block_list` are inputs to the float_to_float16 tool so their names are not clear when used in this pass.

Reply:
Not related to this PR but we should also remove `force_fp16_inputs` from `optimization_options` (which is used for fusion options only) and move it to pass config level. 
It is not used by the `optimize_model` code. Only by the fp16 conversion tool https://github.com/search?q=repo%3Amicrosoft%2Fonnxruntime%20force_fp16_inputs&type=code.